vignettes/RUVIIIPRPS_TCGA_BrCa_RNAseq.Rmd

```diff
@@ -493,12 +532,13 @@ for(i in categorical.variables){
 
 ```
 
-
+**NOTE to improve `plotSilhouette()`**:
+Change the function so that when `plot.output = TRUE`, we do not plot the barplots for each assay separately.
```

Comment:
Need to update `plotSilhouette()` to avoid plotting the barplots for each assay separately.

Reply:
thanks, I have fixed.